karma-conf.js

```diff
@@ -116,8 +116,14 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['PhantomJS'],
+    browsers: ['optchrome'],
```

Comment:
PhantomJS 2.5 will support ES 6, the beta was released on Sunday:
https://groups.google.com/forum/#!topic/phantomjs/AefOuwkgBh0